sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -2053,12 +2053,18 @@ static void _event_cb(netdev_t *dev, netdev_event_t event)
         gnrc_pktsnip_t *pkt = NULL;
         switch (event) {
 #if IS_USED(MODULE_GNRC_IPV6_NIB)
```

Comment:
The #ifdef can now be removed

Reply:
`gnrc_ipv6` is still required to be compiled for this. Otherwise, `gnrc_ipv6_nib` will have no value, but it can be a C-conditional now :-).